release-team/role-handbooks/bug-triage/README.md

```diff
@@ -55,233 +33,208 @@ Bug Triage has a lower time requirement than most roles at the beginning of the
+On the day of the Code Freeze, your responsibility is to try to help contributors to get the approval on their PRs and needed label. Check [How To Escalate](#how-to-escalate) part of the document for guide how to do this.
 
-This is enforced via test-infra automation, as a means of "new release coming in a few weeks - we're only merging relevant PRs in".
+When the code freeze starts, the highest priority has the PRs which had `approved` and `lgtm` labels before the code freeze started and are in the milestones. Depending on the merge queue length, it might be proposed to hold PRs that have `approved` and `lgtm`, but are not in the milestone.
 
-As explained in sections [Code Freeze Preparation](#code-freeze-preparation) and [Filtering](#filtering), it becomes incrementally more difficult to merge new stuff as time progresses. As such, in the days around the start of Code Freeze, owners of issues and PRs should be poked again, reminding them that Code Freeze starts/started at DD/MM.
+You can monitor PRs using the following queries:
+- [PRs supposed to be in the merge pool (`is:pr is:open milestone:v1.17 label:approved label:lgtm -label:do-not-merge/hold`)](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+milestone%3Av1.17+label%3Aapproved+label%3Algtm+-label%3Ado-not-merge%2Fhold)
+- [PRs that have `approved` and `lgtm`, but are on-hold (`is:pr is:open milestone:v1.17 label:approved label:lgtm label:do-not-merge/hold`)](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+milestone%3Av1.17+label%3Aapproved+label%3Algtm+label%3Ado-not-merge%2Fhold+)
+- [PRs that doesn't have `approved` but have `lgtm` (`is:pr is:open milestone:v1.17 -label:approved label:lgtm`)](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+milestone%3Av1.17+-label%3Aapproved+label%3Algtm)
+- [PRs that doesn't have `lgtm` but have `approved` (`is:pr is:open milestone:v1.17 label:approved -label:lgtm`)](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+milestone%3Av1.17+label%3Aapproved+-label%3Algtm+)
 
-The criteria for moving issues are described in [Filtering](#filtering).
+#### Once the Code Freeze starts until one week later
 
-For issues/PRs that look tricky, you can always coordinate with the Release Lead and SIG/PR owners to reach a decision.
+Usually, the removal of issues/PRs doesn't start right once the code freeze starts. Depending on the CI Signal, the merge queue length, and the release team lead's decision, it might be decided to allow some more time for PRs to get merged. At this stage, `kind/bug` PRs are prioritized, but other PRs can be allowed too if there is no risk of destabilizing the release/CI Signal (such PRs require an exception).
```

Comment:
Do we have documented process for PR exceptions?

Reply:
Exceptions are given some treatment in the release lead's [role handbook](https://git.k8s.io/sig-release/release-team/role-handbooks/release-team-lead).  It basically amounts to SIGs' discretion.

Exceptions during freeze also start to bump into cherry-picks.  The [cherry-pick review](https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md#cherry-pick-review) documentation talks about the threshold for these, but again it is largely SIGs discretion.

It would be beneficial to get more clear criteria documented for example in the [Developer Guide page on release](https://git.k8s.io/community/contributors/devel/sig-release/release.md).

Reply:
I thought that we had a pretty hard stop, at least during 1.16, such that after code freeze, everything required an exception and cherry-pick in order to be merged. This paragraph says that some PRs are allowed without exception. It reads to me that PRs which don't destablize the release are allowed without exception, but if the do destablize the release they require exception. I think it is more that any PR that is not a bug requires an exception, and those will only be granted for PRs which don't destablize the release.

Reply:
@josiahbjorgaard Yes, after the code freeze ends, everything requires a cherry-pick. As far as I know, you don't need an exception at that stage, but the cherry-pick is required. However, in the code freeze, we allowed PRs to get merged even without the formal exception. For example, we had a few PRs that were not `priority/critical-urgent` but got merged:
* https://github.com/kubernetes/kubernetes/pull/82288
* https://github.com/kubernetes/kubernetes/pull/81876
* https://github.com/kubernetes/kubernetes/pull/82248 (this one is `kubeadm` related and `kind/design`, but it still got merged without an exception, at least as far as I know)

On the other side, we had this PR https://github.com/kubernetes/kubernetes/pull/78547, which required an exception because it was not marked for 1.16 before the code freeze started.

> I think it is more that any PR that is not a bug requires an exception, and those will only be granted for PRs which don't destablize the release.

I think this is how it should be. But as @tpepper said, we should document it somewhere else, like in the developer guide, and then link it here. Also, I'd like to get some opinions from @guineveresaenger on this topic too.

Reply:
I would like the ideal/the goal to be reflected here and approve of the wording.

We want to have this be the standard.

I agree that we could clarify in the dev guide as well - that feel less urgent to me, because we're only talking about a tiny part of the contributor/release cycle.